client/src/components/lessoncard/LessonCard.css

```diff
@@ -0,0 +1,48 @@
+.lesson-card {
+    display: flex;
+    width: calc(20% - 5px);
+    margin-bottom: 20px;
+    background-color: #ffffff;
+    border-radius: 40px;
+    padding: 30px;
+    box-shadow: 0px 4px 6px rgba(0, 0, 0, 0.1);
+}
+
+.lesson-card.selected {
+    background-color: #cef4fb;
+}
+
+.lesson-image {
+    width: calc(40% - 5px);
+    height: 150px;
+    background-color: #f0f0f0;
+    border-radius: 20px;
+    overflow: hidden;
```

Comment:
forgot to remove this but self reminder to change width to 100% instead of 100vw in Shell.css to fix horizontal overflow issue